src/components/Footer/index.jsx

```diff
@@ -1,64 +1,48 @@
 import React from "react"
 
-import CallToAction from "~/src/components/CallToAction"
+import NewsletterCTA from "../NewsletterCTA"
 import Text from "~/src/components/Text"
-import Logo from "~/src/components/Logo"
-import Location from "./location"
-import Norte2020 from "./Norte2020"
+import Link from "~/src/components/Link"
 import SocialLinks from "./social_links"
 
 import * as styles from "./index.module.css"
 
 function Footer() {
   return (
-    <footer>
-      <div className={styles.blue}>
-        <div className={styles.content}>
-          <div className={styles.logo}>
-            <Logo />
+    <footer className={styles.container}>
+      <NewsletterCTA />
+      <div className={styles.content}>
+        <div className={styles.gridItem}>
+          <div className={styles.itemLabel}>
+            <Text size="large">Go to</Text>
           </div>
-          <div className={styles.callToAction}>
-            <p>
-              <Text>Ready to bring your ideas to life?</Text>{" "}
-              <CallToAction color="white">Let&apos;s talk.</CallToAction>
-            </p>
+          <div className={styles.links}>
+            <Link to="https://subvisual.com/people">People</Link>
+            <Link to="https://jobs.subvisual.com/">Work</Link>
+            <Link to="https://subvisual.com/blog">Blog</Link>
+            <Link to="https://subvisual.com/ventures">Ventures</Link>
           </div>
-          <div className={styles.locations}>
-            <div>
-              <Location
-                align="left"
-                name="Braga, Portugal"
-                image="braga"
-                geoUrl="geo:41.543243,-8.399365"
-                mapsUrl="https://goo.gl/maps/ddvtn1Ez8N72"
-              />
-            </div>
-            <div>
-              <Location
-                align="right"
-                name="Boston, USA"
-                image="boston"
-                geoUrl="geo:42.356742,-71.057583"
-                mapsUrl="https://goo.gl/maps/zouxb7phyLz"
-              />
-            </div>
+        </div>
+        <div className={styles.gridItem}>
+          <div className={styles.itemLabel}>
+            <Text size="large">We&apos;re social</Text>
           </div>
-          <div className={styles.social}>
-            <div className={styles.socialLabel}>
-              <Text size="small">Follow us</Text>
-            </div>
-            <SocialLinks />
+          <SocialLinks />
+        </div>
+        <div className={styles.gridItem}>
```

Comment:
We could add a `display: flex; flex-direction: column; gap: 16px` to the `gridItem`. This way you can get rid off the extra label div and the `margin-bottom`. I think.